config.py

```diff
@@ -41,12 +40,14 @@ def validate_config(config):
     if not (config.allowed_extensions and len(config.allowed_extensions)):
         raise ConfigError("Config error: Allowed extensions can not be empty")
 
-    if not (config.references and len(config.references)):
-        raise ConfigError("Config error: References list can not be empty")
-
-    for ref in config.references:
-        if not all(hasattr(ref, attr) for attr in ['file', 'table', 'local_path_column', 'driver_path_column']):
-            raise ConfigError("Config error: Incorrect media reference settings")
+    if "references" in config:
```

Comment:
Can we somehow simplify these conditions, something like having [] as default if missing key? And then just loop over it...